taskiq-redis/taskiq/result_backend/__init__.py

```diff
@@ -0,0 +1 @@
+"""Redis AsyncResultBackend."""
```

Comment:
Remove this __init__.py since we already have it in taskiq package and don't want it to be overridden.

Reply:
All comments have been made
Check plz